techniques/aria/ARIA11.html

```diff
@@ -22,14 +22,14 @@ <h1>Using ARIA landmarks to identify regions of a page</h1>
 </p>
       <p>Landmarks are inserted into the page using the role attribute on an element that marks the section. The value of the attribute designates the type of landmark. These role values are listed below. For HTML mappings of landmark roles, refer to the <cite>Rules of ARIA attribute usage by HTML element</cite> table in the <a href="https://www.w3.org/TR/html-aria/">ARIA In HTML recommendation</a>.</p>
       <ul>
-         <li><code>banner</code>: A region that contains mostly site-oriented content, such as the logo or a site-specific search tool.</li>
-         <li><code>navigation</code>: A region that contains navigation links to other pages or different parts of the same page.</li>
-         <li><code>main</code>: A region that contains a page's main content.</li>
-         <li><code>region</code>: A region that contains a perceivable section of the page containing content that is sufficiently important for users to be able to navigate to the section. A <code>region</code> landmark isn't exposed as a landmark region unless it has an accessible name, often provided using <code>aria-label</code> or <code>aria-labelledby</code>.</li>
-         <li><code>form</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing. A <code>form</code> landmark isn't exposed as a landmark region unless it has an accessible name.</li>
-         <li><code>search</code>: A region of the page containing search functionality.</li>
-         <li><code>complementary</code>: Any section of the document that supports the main content, yet is separate and meaningful on its own.</li>
-         <li><code>contentinfo</code>: A region that contains information about the parent document such as copyrights and links to privacy statements.</li>
+         <li><code class="language-html">banner</code>: A region that contains mostly site-oriented content, rather than page-specific content.</li>
+         <li><code class="language-html">navigation</code>: A region that contains a collection of navigational elements (usually links) for navigating the document or related documents.</li>
+         <li><code class="language-html">main</code>: A region that contains the main content of a document.</li>
+         <li><code class="language-html">region</code>: A region that contains a perceivable section of the page containing content that is relevant to a specific, author-specified purpose and sufficiently important that users will likely want to be able to navigate to the section easily. A <code class="language-html">region</code> landmark isn't exposed as a landmark region unless it has an accessible name.</li>
+         <li><code class="language-html">form</code>: A region that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing.</li>
```

Comment:
is it also worth mentioning that a `form` is also not meant to be exposed as a landmark unless it is given a name - since this is like `region` and it that was called out for that role?